randovania/games/fusion/logic_database/Sector 4 AQA.txt

```diff
@@ -1971,17 +1971,18 @@ Extra - room_id: [42]
   > Event - Serris
       All of the following:
           Any of the following:
-              # Serris Weapon Requirements
-              Charge Beam or Missiles ≥ 10
+              # Serris Weapon Requirements (50HP)
+              Charge Beam or Missiles ≥ 15
               Missiles ≥ 5 and 20 DMG Missiles
```

Comment:
IMO:
- 10 dmg + 20
- 20 dmg + 10
- combat advanced + 
  - 10 dmg + 10
  - 20 dmg + 5
  
 basically what i did for box.